### PR TITLE
Qwilt open caching CDN

### PR DESCRIPTION
**Pull request by qwilt-igalm**
Details about Qwilt node

### CODE REVIEW

**Comment by amr-mokhtar** [comment]
Because this contribution is an app, it should be placed under `/applications` folder.

File: applications/qwilt/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Installing Qwilt CDN application using helm
+
+Run the following commands to deploy Qwilt CDN application through helm:
+
+`helm install qwiltcdn ./helm/qwiltcdn`
+
+ Sample Output would look like:
+
+> `helm install qwiltcdn /root/openness-20.06/helm/qwiltcdn`
```
Comment (amr-mokhtar) [inline applications/qwilt/README.md:49]: Editorial: snippets of shell inputs & outputs are better bound with ` ```shell ``` ` block.

File: applications/qwilt/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Intel Corporation
```
[the remark this answers is not in the document]
Comment (qwilt-igalm) [reply applications/qwilt/Chart.yaml]: I fixed to Qwilt - please review

File: applications/qwilt/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2020 Actian Corporation
```
[the remark this answers is not in the document]
Comment (qwilt-igalm) [reply applications/qwilt/README.md]: Fixed - please review

File: applications/qwilt/values.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Intel Corporation
+
+PersistentVolume:
+  rootfs:
+    size: "200Gi"
+    path: "/home/var/vd/vol0"
+    imgSrc: "xxx.img"
```
[the remark this answers is not in the document]
Comment (qwilt-igalm) [reply applications/qwilt/values.yaml:8]: It was decided that the users will be directed to Qwilt in order to get the link to update images, please discuss with Mark what exactly is expected to understand which values are required in the yaml file

File: applications/qwilt/templates/NOTES.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Intel Corporation
```
[the remark this answers is not in the document]
Comment (qwilt-igalm) [reply applications/qwilt/templates/NOTES.txt]: Done

File: applications/qwilt/README.md (hunk 1 above)
Comment (qwilt-igalm) [reply applications/qwilt/README.md:50]: Done, please review

**Review by qwilt-igalm — COMMENTED** [review]
I applied the changes

File: applications/qwilt/values.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (amr-mokhtar) [reply applications/qwilt/values.yaml:8]: That's fine. It worth adding that note in the README so that users get familiar how this value can be obtained.

**Pull request merged**